components/dashboard/src/projects/NewProject.tsx

```diff
@@ -80,7 +80,12 @@ export default function NewProject() {
                     )}
                 </Subheading>
 
-                <div className="mt-12">
+                <div className="mt-12 flex space-x-2">
+                    <a href={`/projects/${Project.slug(project)}/settings`}>
```

Comment:
I would expect this button to turn on prebuilds and maybe then take me to the settings to tweak it. Otherwise, maybe replace the text with "Settings"? 

![Recording 2023-09-14 at 12 39 10](https://github.com/gitpod-io/gitpod/assets/29888641/f1d3e74d-0218-422a-acce-cf1f543d198b)

Reply:
Thanks @filiptronicek, you are referring to this view:
<img width="778" alt="Screenshot 2023-09-13 at 15 33 27" src="https://github.com/gitpod-io/gitpod/assets/914497/16bac0c5-7e0d-4c2b-8f1c-65b885d70206">

This is change in UX we're deliberate making in a series of PRs. Decoupling webhook creation from project creation comes requires more guidance, so that one can get an idea of what to do next. @gtsiolis is going to pick this up, but in the meantime we want to ship a 🛹  so that there is at least one CTA to continue with.